code/modules/reagents/reactions/instant/vox.dm

```diff
@@ -0,0 +1,67 @@
+/decl/chemical_reaction/protoslurry
+	name = "Vox Protoslurry"
+	id = "voxslurry"
+	result = "protoslurry"
+	result_amount = 4
+	required_reagents = list(
+		"iron" =    1,
+		"carbon" =  1,
+		"protein" = 2
+	)
+	catalysts = list("voxenzyme" = 5)
+
+/decl/chemical_reaction/vox_medication
+	name = "Vox Medication"
+	id = "vox_medication"
+	result = "voxmeds"
+	result_amount = 2
+	required_reagents = list(
+		"voxslurry" = 3,
+		"voxenzyme" = 1
+	)
```

Comment:
Won't the slurry produced by the first reaction immediately react with the catalyzing enzyme to produce voxmeds?